### ISSUE TITLE
Review: open_MODEX-base_scenario

### ISSUE THREAD

**Issue by jh-RLI** [OpenEnergyPlatform/data-preprocessing#71]
## Issue description

I'm submitting an open dataset with a corresponding OEMetadata string for review. 

The oedatamodel is a collection of multiple tables that are described in the resources. open_MODEX uses the oedatamodel to store input and output related data in the oedatamodel tables. The tables are used as a shared homogonous data source for the framework comparison.

## Workflow checklist

1. GitHub
- [x] I have submitted this issue to have metadata and data review documented (Issue #71 )
- [x] Create a new review-branch and push OEMetadata to new branch (review/open_MODEX-base_scenario#71)

2. OEP
- [x] Upload data to the OEP in schema model_draft (see upload tutorial)
- [ ] Link URL of data in this issue (model_draft.project_nameofdata)

3. Start a Review
- [x] Start a pull request (PR) from review-branch to master
- [x] Assign a reviewer and get in contact

4. Reviewer section
- [x] A reviewer starts working on the issue
- [x] Review data license
- [x] A reviewer finished working on this issue (and awarded a badge)
- [x] Update metadata on table
- [ ] Data moved to its final schema
- [x] Add OEP tags to table
- [ ] Merge PR and delete review-branch
- [ ] Document final links of metadata and data in issue description
- [ ] Close issue

## Metadata and data for review

Here are the links to my data and metadata. Naming follows the pattern model_draft.project_nameofdata:
**Metadata:** https://github.com/OpenEnergyPlatform/data-preprocessing/blob/review/project_nameofdata/data-review/project_nameofdata.json
**Data:** https://openenergy-platform.org/dataedit/view/model_draft/project_nameofdata

## Reviewed and published metadata and data

Final naming and location of the data and metadata after the review are as follows:
**schema.tablename**
* [Metadata](https://github.com/OpenEnergyPlatform/data-preprocessing/blob/review/open_MODEX-base_scenario%2371/data-review/open_MODEX_base_scenario_frame_datapackage.json)
* [Data]()

**Comment by christian-rli** [comment]
metadata can now be parsed by omi. content looks good. (When) will there be any data uploaded to the oep?

**Comment by jh-RLI** [comment]
The upload is in progress. ATM we only have some incomplete data sets on the OEP. There are still some minor data-related issues.

**Comment by christian-rli** [comment]
I noticed that the json covers several resources. However not all 4 tables that can be found [when searching modex](https://openenergy-platform.org/dataedit/view/model_draft?query=modex) on the OEP are covered. Specifically oed_data is missing. At the same time, the three resources are listed in the metadata of each individual table, which I think is technically wrong. The tables might belong together, but the table description should reference only values of the table it describes. Contentwise everything looks ok, as far as i can tell.

**Comment by christian-rli** [comment]
It's more complicated than I first thought, actually. There's a separate table for values, it appears. It is named oed_scalar and has resource descriptions in the metadata. However the table oed_data matches the resource description under `oed_concrete_scalar`. This is a bit messed up. I could see the point of having several resource descriptions under one table, if all tables belong together, but this simply isn't accurate.

**Comment by christian-rli** [comment]
I cannot just download and reupload a new table, because i"m missing the column descriptions to create the new table (due to the same faulty metadata). I also have no time to write new column descriptions for the upload until tomorrow, because my train just arrived. I recommend showing the tables as is in the current directory. They have metadata that is largely correct. Fixing the structure is beyond what I can deliver today. Let me know if I can still help tomorrow forenoon @Ludee . Maybe @jh-RLI can either do a quick fix, or explain why this makes sense the way it is.

**Comment by Ludee** [comment]
I guess that's the problem with the OEDatamodel. It has two formats and the metadata has to fit both.
Thanks for your help!

**Comment by jh-RLI** [comment]
That's right, we used the oed-concrete format to create the metadata because that's the format a user should apply. The missing table only appears in the oed-normalization format. The normalization format is meant to be used on the database. 
After creating the metadata, I didn't think to adjust the resources. I will adjust the resources to the normalization format. @Ludee @christian-rli

**Comment by jh-RLI** [comment]
Should be [fixed](https://github.com/OpenEnergyPlatform/data-preprocessing/blob/15ff5f6bdd783a72179127e89d73a9bc3a47d4d1/data-review/open_MODEX_base_scenario_frame_datapackage.json#L820-L1099).
